packages/bpk-component-button/src/bpk-button.scss

```diff
@@ -38,10 +38,6 @@
     @include bpk-button--destructive;
   }
 
-  &--selected {
-    @include bpk-button--selected;
```

Comment:
Note that this (and any associated tokens) still needs to be cleaned up - it would mean a breaking change in `bpk-mixins` which is undesirable. Perhaps we could mark them as deprecated or hide them in the sassdoc?

Reply:
You're right - the reason I left them alone was because I thought `bpk-component-pagination` was still using the mixins, but I've checked and now I see they're only using the button tokens, not the mixins.

Deprecating them sounds like a good idea, I'll do that.

Reply:
@matthewdavidson I've marked the `selected` button as deprecated.